swagger.yaml

```diff
@@ -2332,6 +2324,108 @@ components:
         - CHANGE_PROPOSAL_STATUS_CHANGED
         - COMMENT_LEFT
 
+    TaskAssignedNotificationData:
+      type: object
+      description: Additional data for the TASK_ASSIGNED notification
+      required:
+        - taskId
+        - taskName
+      properties:
+        taskId:
+          description: ID of the task that was assigned
+          type: string
+          format: uuid
+        taskName:
+          description: Name of the task that was assigned
+          type: string
+
+    TaskStatusChangesNotificationData:
+      type: object
+      description: Additional data for the TASK_STATUS_CHANGED notification
+      required:
+        - taskId
+        - status
+        - taskName
```

Comment:
I added taskName property so that maybe it can be included into notification text without doing a new request. Is this field needed for the notification?

Reply:
I agree task name is needed